vivo-project/VIVO
# Create initial format style enforcement

- All rules are initially suppressed

Resolves: https://jira.duraspace.org/browse/VIVO-1692

# What does this pull request do?
Adds checkstyle Maven tooling. Initially, the checkstyle rules are suppressed.

# How should this be tested?
The build should work as normal (no change).
However, in order to test that the checkstyle machinery works, you can comment out any of the suppression rules, such as:
https://github.com/vivo-project/Vitro/compare/develop...awoods:vivo-1692?expand=1#diff-d3455d71ca9aa05152222bc3d6ae20ddR19
..the build should fail with checkstyle errors.

# Interested parties
@VIVO-project/vivo-committers
